### ISSUE TITLE
Ordered lists serialising/deserialising incorrectly

### ISSUE THREAD

**Issue by sam-jg** [react-page/react-page#504]
I have some across an issue with editing ordered lists. It is hard to describe so here is a gif of it. 
[![https://gyazo.com/4c321b29d3769517b0179a9b48ec3e91](https://i.gyazo.com/4c321b29d3769517b0179a9b48ec3e91.gif)](https://gyazo.com/4c321b29d3769517b0179a9b48ec3e91)

First I have created a list and saved it in our system, then refreshed the page to load that saved version. I then add a new item at the end by pressing 'enter' and press 'tab' to indent it. As you can see in the gif, the text for that new item is moved to the top.

When I press 'tab' I get two validateDOMNesting error's.
`<p> cannot appear as a descendant of <p>`
`<li> cannot appear as a descendant of <li>`

I think it is something to do with the serialising or deserialising because there are no issues with creating the initial list.

**Comment by arekkas** [comment]
Hm, that's strange, and most likely a bug in `slate-edit-list` which has been updated to 0.10.2 recently. However, one issue with upgrading to that version is that there has been a ton of stuff that changed in recent slate versions without backwards compatibility. I didn't had the time yet to review those and upgrade the dependency here, and would really welcome any help I can get for that. Once that's sorted out, we'll sort this one out as well!

**Comment by HriBB** [comment]
I have a similar problem, when list item text is wrapped in a paragraph. Not that I am using this fix to get around #498 

```javascript
const getContent = (state) => {
  const json = JSON.stringify(state);
  const search = '"type":"paragraph"';
  const fix = '"type":"PARAGRAPH/PARAGRAPH"';
  const content = json.replace(new RegExp(search, 'g'), fix);
  console.log(`export default ${content}`)
  return JSON.parse(content)
}
```

**Comment by macrozone** [comment]
can you check if this still is a problem with the latest version? slate was massively updated/rewritten. List handling is still a bit tricky though, so please check if it works properly for you. any feedback is appreciated!
